### PR TITLE
Generate ./hack/README.md

### PR DESCRIPTION
**Pull request by kron4eg**
Signed-off-by: Artiom Diomin <kron82@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/meta.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### This README.md generator
+###
+### That generates README.md with all scripts from this directory described.
+
+README="$(dirname "$0")/README.md"
```
Comment (xrstf) [inline hack/meta.sh]: Can you please make this script consistent with our others, i.e. `cd` into a stable directory and `set -euo pipefail`?

**Review by xrstf — CHANGES_REQUESTED** [review]
Minor nitpick

File: hack/meta.sh (hunk 1 above)
Comment (kron4eg) [reply hack/meta.sh]: 👍

**Review by xrstf — APPROVED** [review]
/approve

**Review by xrstf — APPROVED** [review]
/approve

**Review by xmudrii — APPROVED** [review]
/lgtm

**Pull request merged**